mycrt-backend/src/server.py

```diff
@@ -107,21 +108,64 @@ def get_rds_instances():
 
     return jsonify({'status': response['ResponseMetaData']['HTTPStatusCode'], 'error': response['Error']['Code']})
 
-@app.route('/login-test', methods=['PUT'])
+@login_manager.request_loader
+def load_user_from_request(request):
+    auth = request.authorization
+    if auth:
+        username = auth.username
+        password = auth.password
+        user = user_datastore.find_user(username=username)
+        if user is not None and verify_password(password, user.password):
+            return user
+    return None
+
+@app.route('/login', methods=['POST'])
 def login():
-    user_repository.register_user(username="test-user", password="test123", email='test@test.com', access_key="test_access_key", secret_key="test_secret_key")
-
+    user = current_user
+    if user is not None:
+        login_user(user)
+        return jsonify({"status" : "success",
+            "user" : {
+                "username" : user.username,
+                "password" : user.password,
+                "access_key" : user.access_key,
+                "secret_key" : user.secret_key,
+                "email" : user.email
+                }
+            }), 200
+    else:
+        return jsonify({"status" : "fail",
+            "user" : {}
+            })
 
 @login_required
 @app.route('/logout', methods=['POST'])
 def logout():
     login_manager.logout_user(current_user)
     return redirect('/', code=302)
 

```

Comment:
I thought I was working on register? Or did you want to go ahead and commit this?

Reply:
You can make changes to it if you want but it is basically done. I remember saying I would do it like a week ago?

Reply:
Sure I can go ahead and add the error checking and anything else since I have it done already